tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreRuntimeVersion80Tests.cs

```diff
@@ -0,0 +1,76 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Oryx.BuildScriptGenerator;
+using Microsoft.Oryx.BuildScriptGenerator.Common;
+using Microsoft.Oryx.BuildScriptGenerator.DotNetCore;
+using Microsoft.Oryx.BuildScriptGeneratorCli;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "dotnetcore-8.0")]
+    public class DotNetCoreRuntimeVersion80Tests : DotNetCoreEndToEndTestsBase
+    {
+        public DotNetCoreRuntimeVersion80Tests(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
+            : base(output, testTempDirTestFixture)
+        {
+        }
+
+        [Theory]
+        [Trait("build-image", "github-actions-debian-buster")]
```

Comment:
will fix this to bookworm in later PR, do not want to hold up the release for this minor change